src/frontend/components/Felleskomponenter/Pensjonsmodal/språkUtils.ts

```diff
@@ -1,16 +1,41 @@
+import { LocaleRecordBlock } from '../../../typer/common';
 import { PersonType } from '../../../typer/personType';
 import { ESanitySteg, ISanitySpørsmålDokument } from '../../../typer/sanity/sanity';
 import { ITekstinnhold } from '../../../typer/sanity/tekstInnhold';
 
-export const mottarPensjonNåFeilmeldingSpråkId = (personType: PersonType): string => {
+export const mottarPensjonNåFeilmelding = ({
+    personType,
+    gjelderUtland,
+    tekster,
+}: {
+    personType: PersonType;
+    gjelderUtland: boolean;
+    tekster: ITekstinnhold;
+}): LocaleRecordBlock => {
+    const eøsForBarnTekstinnhold = tekster.EØS_FOR_BARN;
+    const eøsForSøkerTekstinnhold = tekster.EØS_FOR_SØKER;
+
     switch (personType) {
         case PersonType.andreForelder:
-            return 'ombarnet.andre-forelder.pensjonnå.feilmelding';
+            if (gjelderUtland) {
+                throw Error('pensjonUtlandAndreForelder ikke implementert');
```

Comment:
Tror ikke det skal være mulig at vi havner i denne tilstanden, men tenkte det var betre at vi kasta en feilmelding enn at vi viste teksten for pensjonNorgeAndreForelder i dette tilfellet

Reply:
Sett!

Reply:
Fikset i [Legg til tekster for alle tilfeller](https://github.com/navikt/familie-ks-soknad/pull/99/commits/93dfbcd97f4690587b32bc9f29b629bba4bc752d)